src/bootstrap/compile.rs

```diff
@@ -574,7 +574,9 @@ fn copy_sanitizers(
             || target == "aarch64-apple-darwin"
             || target == "aarch64-apple-ios"
             || target == "aarch64-apple-ios-sim"
+            || target == "aarch64-apple-ios-catalyst"
```

Comment:
These don't look like the right target triples for Rust or LLVM. Shouldn't they be `$arch-apple-ios-macabi`?

Reply:
Oh yes, thanks, this is what we call the thing in chromium's build system and I typoed that in here.

Reply:
This fact is reflected in the changes here: [src/bootstrap/llvm.rs](https://github.com/rust-lang/rust/pull/115644/files#diff-67761c315d694c57b071f47f7f2323d4260d7e161ea861511c5ee3e8c1d8e277)

The name of the runtime is "osx", not "iosmacabi" or something like that.